apps/omg_eth/lib/omg_eth/ethereum_height.ex

```diff
@@ -45,16 +45,9 @@ defmodule OMG.Eth.EthereumHeight do
     {:reply, {:ok, ethereum_height}, ethereum_height}
   end
 
-  def handle_info({:internal_event_bus, :ethereum_new_height, new_height}, state) do
-    case is_integer(new_height) do
```

Comment:
we've identified with @unnawut that there's no way a "non integer term" would get into the omg bus bus